vdsm_hooks/log_firmware/README

```diff
@@ -0,0 +1,42 @@
+log_firmware vdsm hook
+=================================
+This hook appends a VM's firmware to a log file on the host.
```

Comment:
... VM's firmware *log output* to a log file ...

Reply:
Changed to "... VM's firmware debug logging to a log ...". I hope this is clear enough...